### ISSUE TITLE
Windows extension over Collections doesn't work

### ISSUE THREAD

**Issue by Raduan77** [apple/swift-algorithms#81]
I was playing around with your package and decided to take a look at a [Windows example](https://github.com/apple/swift-algorithms/blob/main/Guides/Windows.md). After setting a very simple playground and trying this example, I can not build it. Instead of building it, it throws the next error: 
```sh
/path/to/playground/main.swift:5:32: error: value of type 'String' has no member 'windows'
let windowed: [String] = swift.windows(ofCount: 2)
```
First of all, I thought that there is something on my end, however, other examples work fine(I've tried [this](https://github.com/apple/swift-algorithms/blob/main/Guides/RandomSampling.md) and [this](https://github.com/apple/swift-algorithms/blob/main/Guides/Unique.md))

**Swift Algorithms version:** `0.0.2`
**Swift version:** `Apple Swift version 5.3.2 (swiftlang-1200.0.45 clang-1200.0.32.28)
Target: x86_64-apple-darwin20.3.0`

### Checklist
- [x] If possible, I've reproduced the issue using the `main` branch of this package
- [x] I've searched for [existing GitHub issues](https://github.com/apple/swift-algorithms/issues)

### Steps to Reproduce
Here is my `Package.swift`:
```swift
import PackageDescription

let package = Package(
    name: "playground",
    dependencies: [
        .package(url: "https://github.com/apple/swift-algorithms", from: "0.0.2"),
    ],
    targets: [
        .target(
            name: "playground",
            dependencies: [
                .product(name: "Algorithms", package: "swift-algorithms"),
            ],
            path: "Sources/playground"
            ),
    ]
)
```
and `main.swift`:
```swift
import Foundation
import Algorithms
let swift = "swift"

let windowed: [String] = swift.windows(ofCount: 2) 
print(windowed)
```

### Expected behavior
I expected to see the next result: `["sw", "wi", "if", "ft"]`

### Actual behavior
I get the error after running `swift build`: 
```sh
/path/to/playground/main.swift:5:32: error: value of type 'String' has no member 'windows'
let windowed: [String] = swift.windows(ofCount: 2)
```

**Comment by timvermeulen** [comment]
Version 0.0.2 was released in October, and it did not yet include the `windows(ofCount:)` method. It will be included in the upcoming 0.0.3 release, and if you'd like to start using it right now, you can already depend on the `main` branch instead:

```swift
.package(url: "http://github.com/apple/swift-algorithms", .branch("main"))
```

**Comment by Raduan77** [comment]
Thanks for the clarification, it makes sense.
